src/plugins/intel_cpu/src/nodes/executors/subgraph.hpp

```diff
@@ -89,7 +89,7 @@ class SubgraphBaseExecutor {
     size_t m_internal_buffer_size = 0;
     size_t m_tensor_rank = 0;
 
-    const size_t rank6D = 6;
+    static const size_t rank6D;
```

Comment:
I'd rather have it `static constexpr`. It will help to avoid initialization in .cpp (in-place initialization should work here)